README.md

```
@@ -20,12 +20,12 @@ To use the package, simply import it:
 import mathsys
 ```
 
-It is recommended that you import a version especifically, the only version available right now is `dev`:
+It is recommended that you import a version specifically, the only version available right now is `dev`:
 ```py
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)